scripts/entrypoint.sh

```diff
@@ -1,9 +1,25 @@
 #! /usr/bin/env bash
 
 /usr/bin/redis-server /etc/redis/redis.conf
-service mongodb start
+(mongod & ) 1>/dev/null
```

Comment:
The newer version of mongo expects to be ran as a service using systemd rather than upstart. Unfortunately systemd doesn't immediately work with docker. I had already spent longer than I wanted on this PR so this was the quick and dirty solution.